### PR TITLE
[KMS] Enabling Terraform tests for KMS.

### PR DESCRIPTION
**Pull request by taras-kobernyk-localstack**
Our test suite executes tests that exactly match the names specified in the `tests/terraform/terraform-tests.yaml`. The current config lists list names of test bundles instead of tests themselves, so no tests match those names, nothing gets executed.

This PR lists full names of the tests instead.

### CODE REVIEW

File: tests/integration/test_kms.py [hunk 1]
```diff
@@ -267,12 +267,48 @@ def test_sign_verify(self, kms_client, key_type, kms_create_key):
...
+
+        signature = kms_client.sign(MessageType="RAW", **kwargs)["Signature"]
+        with pytest.raises(kms_client.exceptions.KMSInvalidSignatureException):
+            assert not kms_client.verify(MessageType="DIGEST", Signature=signature, **kwargs)[
+                "SignatureValid"
+            ]
+
+        signature = kms_client.sign(MessageType="DIGEST", **kwargs)["Signature"]
+        with pytest.raises(kms_client.exceptions.KMSInvalidSignatureException):
+            assert not kms_client.verify(MessageType="RAW", Signature=signature, **kwargs)[
```
Comment (dominikschubert) [inline tests/integration/test_kms.py:304]:
```suggestion
            kms_client.verify(MessageType="RAW", Signature=signature, **kwargs)[
```

File: tests/integration/test_kms.py [hunk 2]
```diff
@@ -267,12 +267,48 @@ def test_sign_verify(self, kms_client, key_type, kms_create_key):
...
+        # supposed to fail.
+        #
+        # There is a possibility that I do not understand how digests work, so can't write better tests. If we have
+        # any issues with the digests - know that the current approach is not a calculated design, but rather a guess.
+        signature = kms_client.sign(MessageType="RAW", **kwargs)["Signature"]
+        assert kms_client.verify(MessageType="RAW", Signature=signature, **kwargs)["SignatureValid"]
+
+        signature = kms_client.sign(MessageType="RAW", **kwargs)["Signature"]
+        with pytest.raises(kms_client.exceptions.KMSInvalidSignatureException):
+            assert not kms_client.verify(MessageType="DIGEST", Signature=signature, **kwargs)[
```
Comment (dominikschubert) [inline tests/integration/test_kms.py:298]:
```suggestion
            kms_client.verify(MessageType="DIGEST", Signature=signature, **kwargs)[
```

**Review by dominikschubert — APPROVED** [review]
minor nit, otherwise LGTM :+1: 

Love the additional comments :+1:

File: tests/integration/test_kms.py (hunk 1 above)
Comment (taras-kobernyk-localstack) [reply tests/integration/test_kms.py]: My bad. This part from a commit that was not supposed to be a part of this PR. Removed the commit.

**Pull request merged**